src/pages/console/adminui/access-management.mdx

```diff
@@ -3,7 +3,7 @@ export const meta = {
   description: `Manage team access to a project`,
 };
 
-With Amplify Studio, team members with different job functions can collaborate on different aspects of a project deployed in Amplify. Studio developers can create accounts with scoped access to resources and invite team members to join via email. Team members with *Full access* can create and manage AWS resources, while team members with *Manage only* access can edit application content and users.
+With Amplify Studio, team members with different job functions can collaborate on different aspects of a project deployed in Amplify. Studio developers can create accounts with scoped access to resources and invite team members to join via email.
```

Comment:
This is slightly inaccurate - customers can add "manage only" users, but cannot scope their access to specific resources. For example, if I want to provide manage-only access to Users, but not to Data Manager, I cannot scope that. I think it was more accurate in the original statement.

Reply:
removing it here as in the first section we also provide the following

```
Full access allows the team member to create and manage AWS resources.
Manage only access allows the team member to edit app content and users.
```
felt this was duplicated information with no formatting

Reply:
wesley, we will need to improve the wording for sure. but the commit here is removing it as its duplicated